cassandane/Cassandane/TestEntity/Factory/Mailbox.pm

```diff
@@ -0,0 +1,52 @@
+package Cassandane::TestEntity::Factory::Mailbox;
+use Moo;
+
+with 'Cassandane::TestEntity::Role::Factory';
+
+use feature 'state';
+
+require Cassandane::TestEntity::Instance::Mailbox;
+
+sub datatype { 'Mailbox' }
+
+sub instance_class  { 'Cassandane::TestEntity::Instance::Mailbox' }
+
+sub fill_in_creation_defaults {
+    my ($self, $prop) = @_;
+
+    state $i = 1;
+    $prop->{name} //= 'Mailbox #' . $i++;
+
+    return;
+}
+
+sub inbox {
+    my ($self) = @_;
+    my $dt = $self->datatype;
+
+    my $jmap = $self->user->jmap;
+    delete $jmap->{CreatedIds};
```

Comment:
This probably deserves a comment. Or something.

Reply:
My feeling is that probably we should nerf this CreatedIds-tracking behavior, but I need to learn more.  For now, a comment.  (This will go away if we switch to JMAP::Tester.)